src/features/SendAssets/SendAssets.reducer.tsx

```diff
@@ -125,7 +131,8 @@ sendAssetsReducer.actionTypes = {
   REQUEST_SEND: 'REQUEST_SEND',
   SEND_SUCCESS: 'SEND_SUCCESS',
   REQUEST_RESUBMIT: 'REQUEST_RESUBMIT',
```

Comment:
Can we remove the current resubmit functionality if we are doing it another way?

Reply:
This is functionality is attached to a button already i think. This pr is just used to add query params. I've changed the title of it to reflect that better. For separation of concerns, we can do the tx receipt resubmit logic + button in another pr.